src/properties.rs

```diff
@@ -31,6 +31,7 @@ where
             DisplaySize::Display128x32 => (0, 0),
             DisplaySize::Display96x16 => (0, 0),
             DisplaySize::Display72x40 => (28, 0),
+            DisplaySize::Display64x48 => (32, 0),
```

Comment:
How did you arrive at this offset value?

Reply:
Trial and error

Reply:
Can you try 0?

Reply:
When it's set to 0 and the below code is run, the left half of the screen is white and the right half is black

```rust
for i in 0..64 { for j in 0..48 {
    disp.set_pixel(i,j,1);
}}
disp.flush().unwrap();
```

Reply:
It is kind of hard to follow what you're doing. Which display are we talking about now? I thought you wanted to fix the TerminalMode first?

Key to getting this fixed is the right combination of configuration values and GraphicsMode works in a totally different way to TerminalMode which is essentially setting the framebuffer values so it can automatically advance from left to right and top to bottom. If it skips to the next line after 4 characters that is typically caused by telling the display driver hardware the wrong display dimensions. It could also be that the TerminalMode calculations are wrong.

Reply:
Ah, sorry, I didn't realize the inquiry was related to the TerminalMode issue. This is still with the 64x48px display. Changing `32` to `0` seems to result in the same output as the above image.

Reply:
Hm, so something else is wrong.